substrate/frame/insecure-randomness-collective-flip/src/lib.rs

```diff
@@ -118,8 +118,7 @@ pub mod pallet {
 	/// is arranged as a ring buffer with `block_number % 81` being the index into the `Vec` of
 	/// the oldest hash.
 	#[pallet::storage]
-	#[pallet::getter(fn random_material)]
-	pub(super) type RandomMaterial<T: Config> =
+	pub type RandomMaterial<T: Config> =
```

Comment:
Alsp here please re-add the getter as a manual implementation so that it is not breaking.  
cchttps://github.com/paritytech/polkadot-sdk/issues/223